naijahacks/certificate-generator
# Implement user data source using json

## Changes
1. Add json data for attendees and teams
2. Change data source from csv to json
3. Implement clean email validation

## How to test
1. Clone the repository
2. Navigate to project directory and run npm install
3. Run npm start, then navigate to http://localhost:3000
4. check assets/json/redactedUsers.json to input any valid email
`e.g muhammad6440@gmail.com`

## Screenshots
![image](https://user-images.githubusercontent.com/24432247/73321183-f80b0280-4241-11ea-8f4b-5326da4ccc99.png)

![image](https://user-images.githubusercontent.com/24432247/73321216-153fd100-4242-11ea-90ee-82a6e31ae422.png)
